tdrs-backend/tdpservice/data_files/admin.py

```diff
@@ -70,6 +99,7 @@ def data_file_summary(self, obj):
         'version',
         'data_file_summary',
         'error_report_link',
+        'rpm_links'
```

Comment:
personal preference - i don't really love these in the list display. i think the related models available in the edit/detail view for both the datafile and the reparse metal model is sufficient

Reply:
Yes, my plan was to discuss this if OH today to get some feedback

Reply:
if we are adding 5 links then I agree with Jan since they overfill the column